### DISCUSSION TITLE
这个开始抢票是在12点准时打开还是提前一分钟

### DISCUSSION THREAD

**Discussion by zhumengqianqiu** [mikumifa/biliTickerBuy#306] [提问专区]
这个开始抢票是在12点准时打开还是提前一分钟（设置配置时间是12点整），提出疑问的原因是有人提前点开始抢票导致异常，那么是否能在开票前一两分钟点击开始抢票，希望大佬捞一下，万分感谢

**Comment by mikumifa** [comment]
看一下#272
